### ISSUE TITLE
Bug: upgrading Docker image from 0.7.2 to 0.7.4 - The 0.7.4 version doesn't work

### ISSUE THREAD

**Issue by Eliastik** [ArchiveBox/ArchiveBox#1393]
#### Describe the bug

When upgrading the Docker image from version 0.7.2 to 0.7.4, the container don't work anymore and crash with an error in the logs.

Downgrading to 0.7.2 fixes this issue.

#### Steps to reproduce

Spawn a container with this docker-compose.yml file:

```
# Usage:
#     docker-compose run archivebox init --setup
#     docker-compose up
#     echo "https://example.com" | docker-compose run archivebox archivebox add
#     docker-compose run archivebox add --depth=1 https://example.com/some/feed.rss
#     docker-compose run archivebox config --set PUBLIC_INDEX=True
#     docker-compose run archivebox help
# Documentation:
#     https://github.com/ArchiveBox/ArchiveBox/wiki/Docker#docker-compose

version: '2.4'

services:
    archivebox:
        # build: .                              # for developers working on archivebox
        image: ${DOCKER_IMAGE:-archivebox/archivebox:0.7.4}
        command: server --quick-init 0.0.0.0:8000
        restart: unless-stopped
        ports:
            - 8000:8000
        environment:
            - ALLOWED_HOSTS=*                   # add any config options you want as env vars
            - MEDIA_MAX_SIZE=750m
            # - CHROME_HEADLESS=False
            # - CHROME_USER_DATA_DIR=/home/archivebox/.config/chromium
            # - SEARCH_BACKEND_ENGINE=sonic     # uncomment these if you enable sonic below
            # - SEARCH_BACKEND_HOST_NAME=sonic
            # - SEARCH_BACKEND_PASSWORD=SecretPassword
        volumes:
            - ./data:/data
            # - ./archivebox:/app/archivebox    # for developers working on archivebox
```

#### Screenshots or log output

The log is : 

```
[i] [2024-03-30 11:04:05] ArchiveBox v0.7.3: archivebox server --quick-init 0.0.0.0:8000
    > /data

[X] Could not find profile "Default" in CHROME_USER_DATA_DIR.
    /data/personas/Default/chromium
    Make sure you set it to a Chrome user data directory containing a Default profile folder.
    For more info see:
        https://github.com/ArchiveBox/ArchiveBox/wiki/Configuration#CHROME_USER_DATA_DIR

    Try removing /Default from the end e.g.:
Traceback (most recent call last):
  File "/usr/local/bin/archivebox", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/app/archivebox/cli/__init__.py", line 140, in main
    run_subcommand(
  File "/app/archivebox/cli/__init__.py", line 74, in run_subcommand
    setup_django(in_memory_db=subcommand in fake_db, check_db=cmd_requires_db and not init_pending)
  File "/app/archivebox/config.py", line 1389, in setup_django
    check_system_config()
  File "/app/archivebox/config.py", line 1297, in check_system_config
    stderr('        CHROME_USER_DATA_DIR="{}"'.format(config['CHROME_USER_DATA_DIR'].split('/Default')[0]))
                                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'PosixPath' object has no attribute 'split'
```

#### ArchiveBox version

0.7.4 (Docker version) on Ubuntu 20.04.6 LTS host

**Comment by pirate** [comment]
Duplicate: https://github.com/ArchiveBox/ArchiveBox/issues/1391

Temporary solution is mentioned in that issue. Should be fixed in a few days.

**Comment by dsander** [comment]
@pirate I am having the same problem as described by the OP, the issue you linked seems to be related to the `:dev` tag or having the `CHROME_USER_DATA_DIR` environment variable set. Am I missing the actual workaround?

**Comment by pirate** [comment]
I fixed the broken Docker tags: `:dev`, `:0.7`, or `:0.7.2` should point to the working previous release again. Sorry for the trouble!
